core/src/main/java/com/cognifide/aem/stubs/core/ConfigurableStubManager.java

```diff
@@ -99,11 +100,13 @@ protected void visit(Resource resource) {
   }
 
   private boolean isScript(String path, Stubs<?> runnable) {
-    return wildcardMatch(path, format("%s/%s/**/*%s", getRootPath(), runnable.getId(), config.scriptExtension()));
+    return StringUtils.startsWith(path, format("%s/%s/", getRootPath(), runnable.getId()))
+      && StringUtils.endsWith(path, config.scriptExtension());
   }
 
   private boolean isMapping(String path, Stubs<?> runnable) {
-    return wildcardMatch(path, format("%s/%s/**/*%s", getRootPath(), runnable.getId(), config.mappingExtension()));
+    return StringUtils.startsWith(path, format("%s/%s/", getRootPath(), runnable.getId()))
```

Comment:
scripts on level /var/stubs/wiremock/hello-world.stub.groovy was not reloaded ;)